std/algorithm/comparison.d

```diff
@@ -63,6 +63,15 @@ of the first matching value in $(D values), or $(D 0) if $(D value)
 is not _among $(D values). The predicate $(D pred) is used to
 compare values, and uses equality by default.
 
+Params:
+    pred = The predicate used to compare the values.
+    value = The value to search.
```

Comment:
Nitpick, but this wording technically means that something is searched for in *value*. Should be "The value to search for."